spec/index.bs

```diff
@@ -7334,6 +7334,22 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
         </div>
 </dl>
 
+<div algorithm>
+    To <dfn abstract-op>get the entry point</dfn>({{GPUShaderStage}} |stage|,
+    {{GPUProgrammableStage}} |descriptor|)
+
+    - If |descriptor|.{{GPUProgrammableStage/entryPoint}} is not `null`:
```

Comment:
Oops one last thing. It's not `null` when omitted - it just isn't in the `map` at all:

```suggestion
    - If |descriptor|.{{GPUProgrammableStage/entryPoint}} is [=map/exists|provided=]:
```